cfme/intelligence/chargeback/rates.py

```diff
@@ -198,8 +198,13 @@ def update(self, updates):
 
     def delete(self):
         # Delete a rate in UI
-        view = navigate_to(self, 'Details')
-        view.configuration.item_select('Remove from the VMDB', handle_alert=True)
+        view = navigate_to(self, 'Details', wait_for_view=True)
+        assert view.is_displayed
+        view.configuration.item_select(
+            VersionPick({
+                Version.lowest(): 'Remove from the VMDB',
+                '5.9': 'Remove selected Chargeback Rates from the VMDB'}),
```

Comment:
This may not be relevant - the configuration dropdown wasn't where it should be for the test, and it should be on details